test/conversion.jl

```diff
@@ -1,103 +1,46 @@
-using Geodesy
-using Base.Test
-
-################################################
-### Helpers for testing approximate equality ###
-################################################
-
-macro type_approx_eq(a, b)
-    quote
-        @test names($(esc(a))) == names($(esc(b)))
-        for n in names($(esc(a)))
-            @test_approx_eq $(esc(a)).(n) $(esc(b)).(n)
-        end
-    end
-end
-
-macro xy_approx_eq(a, b)
-    quote
-        @test_approx_eq getX($(esc(a))) getX($(esc(b)))
-        @test_approx_eq getY($(esc(a))) getY($(esc(b)))
-    end
-end
-macro xy_approx_eq_eps(a, b, eps)
-    quote
-        @test_approx_eq_eps getX($(esc(a))) getX($(esc(b))) $(esc(eps))
-        @test_approx_eq_eps getY($(esc(a))) getY($(esc(b))) $(esc(eps))
-    end
-end
-
-macro xyz_approx_eq(a, b)
-    quote
-        @test_approx_eq getX($(esc(a))) getX($(esc(b)))
-        @test_approx_eq getY($(esc(a))) getY($(esc(b)))
-        @test_approx_eq getZ($(esc(a))) getZ($(esc(b)))
-    end
-end
-macro xyz_approx_eq_eps(a, b, eps)
-    quote
-        @test_approx_eq_eps getX($(esc(a))) getX($(esc(b))) $(esc(eps))
-        @test_approx_eq_eps getY($(esc(a))) getY($(esc(b))) $(esc(eps))
-        @test_approx_eq_eps getZ($(esc(a))) getZ($(esc(b))) $(esc(eps))
+@testset "Co-ordinate system conversion" begin
+    ###################################
+    ### Testing fixed relationships ###
+    ###################################
+
+    lla = LLA(42.3673, -71.0960, 0.)
+    lla_ref = LLA(42.36299, -71.09183, 0.)
+
+    # LLA -> ECEF
+    ecef_ref = ECEF(1529073.1560519305, -4465040.019013103, 4275835.339260309)
+    @xyz_approx_eq ECEF(lla, wgs84) ecef_ref
+
+    #LLA -> ENU
+    enu_ref = ENU(-343.493749083977, 478.764855466788, -0.027242885224325164)
+    @xyz_approx_eq_eps ENU(lla, lla_ref, wgs84) enu_ref 1e-8
+
+    #############################
+    ### Testing random errors ###
+    #############################
+    """
+        randLLA(h_min = -5e6, h_max = 5e6)
+
+    Random LLA coordinate with evenly-distributed angles (according to the
+    Haar measure over the sphere)
```

Comment:
It just means uniformly over a sphere (in terms of surface area). I'll write it that way.